examples/provider.ts

```diff
@@ -0,0 +1,44 @@
+import { DefaultHederaProvider } from "@ethersproject/providers";
```

Comment:
The implementation must be such that users do not import anything except for `ethers`, thus importing `hashgraph/sdk` is not something we want to end up with

Reply:
I think we should solve this in a separate PR, where we have refactored the `BaseProvider` interface